src/main/java/com/exadel/movie_new/service/MovieServiceImpl.java

```diff
@@ -1,20 +1,28 @@
+package com.exadel.movie_new.service;
+
+import com.exadel.movie_new.exception.FilesException;
+import com.exadel.movie_new.exception.MovieNotFoundException;
+import com.exadel.movie_new.model.Movie;
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+import org.springframework.stereotype.Service;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
 import java.nio.file.Path;
 import java.util.ArrayList;
 import java.util.List;
 
 @Service
 public class MovieServiceImpl implements MovieService {
 
-
     @Override
     public Movie addMovie(Movie movie) {
         List<Movie> movieData = readJsonData();
-      
-
         boolean isPresent = false;
         if (movieData != null && !movieData.isEmpty()) {
             isPresent = movieData.parallelStream().anyMatch(mov -> mov.getId().equals(movie.getId()));
-
         } else {

```

Comment:
what if movie with such ID already exists? You should not just return movie because for client it looks like all fine and movie has been added. I would recommend you throw exception like 
MovieAlreadyExist(String.format("movie with such id %s already exists", movie.getId())